roles/technical_principal.md

```diff
@@ -0,0 +1,154 @@
+# Technical Principal
+
+Our Technical Principals are consultative engineers who are commercially responsible for growing Made Tech both in terms of maintaining the quality of our teams as we scale and in helping win new business.
+
+## What does the job entail?
+
+Technical Principals sit across multiple delivery teams ensuring software engineering excellence and engaging in technical discovery work.
+
+Through coaching and mentoring Technical Pricipals will make sure Leads and Seniors are equipped with the skills needed to be successful in delivery. They will be helping grow our team into successful technical leaders and be responsible for helping others become Technical Pricipals.
+
+As an escalation point for software engineering issues, Technical Principals will be on hand when necessary, and to involve the wider engineering team if necessary.
+
+Helping in technical discovery work, Technical Principals will be able to engage in stakeholder workshops, technical feasibility studies, high level roadmapping, and ultimately shape up next steps for a Made Tech delivery team to be engaged.
+
+Technical Principals are expected to be seen as thought leaders in software engineering, to talk about our work in public, to remain technically current and push engineering at Made Tech forward.
+
+## What does software delivery look like at Made Tech?
+
+We primarily write and deliver custom software to our customers. Before a project kicks off we will have a number of sessions where we begin to understand the customer’s business and what they are looking to achieve. We then formulate a roadmap for them, potentially a business case too, before starting delivery.
+
+We build solutions in a variety of languages and platforms, historically Ruby on Rails, we tend to use React, JavaScript, Elixir, and Clojure these days. We will usually be responsible for setting up a customer’s infrastructure on AWS with tools like Ansible and Terraform though sometimes we opt for a Platform as a Service like Heroku. We make sure there is a Continuous Delivery pipeline running our automated test suites before deploying changes.
```

Comment:
This is probably a big feature creep on this PR, but I think it's time to move away from this "historically Rails.." language, as I think it's keeping us anchored. Perhaps:

We build solutions using the best language or tool for the job, which recently may be one or more of Ruby, C#, React, JavaScript, Elixir or Clojure.

Reply:
One of these things isn't the same kind as the others. <React> :eyes:

Reply:
JSX is Craig.

Yes it's creep @chrisblackburn I'll add an issue to remove.

Reply:
See https://github.com/madetech/handbook/issues/195

Reply:
:P JSX is an extension of JS